README.md

```diff
@@ -50,9 +50,13 @@ desired plays.  IE:
 openstack-ansible setup-everything.yml
 ```
 
+**Note**: If you have cloned the os-ansible-deployment repository to anywhere
+other than `/opt/os-ansible-deployment` then you will need to edit the paths
+located in `rpc-extras/playbooks/ansible.cfg` to ensure that the dynamic
+inventory, libraries and roles are accessible.
+
```

Comment:
this is probably better as a step in the 'basic setup' steps.  ie

3. edit playbooks/ansible.cfg and ensure the paths to the roles, playbooks and inventory are correct